documentation/vespa-quick-start-multinode-aws.html

```diff
@@ -43,91 +43,47 @@ <h2 id="launch-instances">Launch instances on <a href="aws.amazon.com/ec2">AWS E
 <h2 id="install-vespa">Install and run Vespa</h2>
 
 <p> Create a hostfile listing the hosts in the system
```

Comment:
Move the section of creating a host file to after ssh'ing into the Vespa admininstration EC2 instance.